packages/extension-commerce-bm-seo/config/default.json

```diff
@@ -1,5 +1,6 @@
 {
     "enabled": true,
+    "routingMode": "api_first",
```

Comment:
I noticed that this project is missing a sample.json file. In the future we are going to be using sample.json during project generation with presets. You might want to add it now vs having to comeback and add it later. I think there is a blurb about the 2 files in the readme in this folder.